### DISCUSSION TITLE
HELLO

### DISCUSSION THREAD

**Discussion by AsimoRoblox** [community/community#38380] [Other Feature Feedback, Questions, & Ideas]
good morning I have a question in visual studio code html language how to do a transition between pages?

**Comment by jfullstackdev** [comment]
what do you mean by `transition between pages` ? 

once you open each file, it's just like Chrome tab where you can switch from time to time 

![image](https://user-images.githubusercontent.com/47092464/200153352-e3627918-cdc9-4e96-8269-aac88d5fdb8d.png)
